subsys/net/l2/ethernet/gptp/gptp_mi.c

```diff
@@ -734,16 +734,22 @@ static void gptp_update_local_port_clock(void)
 {
 	struct gptp_clk_slave_sync_state *state;
 	struct gptp_global_ds *global_ds;
+	struct gptp_servo_ds *servo_ds;
 	struct gptp_port_ds *port_ds;
 	int port;
-	int64_t nanosecond_diff;
-	int64_t second_diff;
+	uint64_t sync_receipt_ts;
+	uint64_t sync_receipt_local_ts;
```

Comment:
Aren't these variables that exist in the standard (here or elsewhere)? If so, it would be great if you could point to the standard (standard, version, clause, optionally table/figure) and ensure that you use the standardized names adjusted for Zephyr naming conventions only.

It's unfortunate (and has been fixed by myself on many occasions in other contexts - just recently for parts of (g)PTP as well) that we do not systematically point to the standard and use standard names for structs, variables and functions where we implement/use standard primitives. This makes it harder than necessary to add standard features to existing code w/o a structural clean-up.

Of course it's not your responsibility to fix existing code in that respect (although it would be helpful and can often be done on-the-fly with little effort).

Reply:
> * Please ensure that whenever you name variables you use names from the standard (and ideally point to the section in the standard where they are defined). Otherwise it's very hard to check the correctness of the proposed algorithm. Not using well-defined names from the standard makes it so much harder than necessary to review and understand the code and add standard-based features in the future.
> 

This is very important point, we should keep the names mentioned in the spec and in the code as close as possible. 

> 
> Of course it's not your responsibility to fix existing code in that respect (although it would be helpful and can often be done on-the-fly with little effort).

If something is wrongly named in the code, please consider sending a PR to fix it.